client/src/components/NotificationEntry.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const Note = ({ notification, clickHandler, index}) => {
+  const style = {
```

Comment:
These customised styles need to be moved into our css file in html <link> later.